src/seedu/addressbook/AddressBook.java

```diff
@@ -721,7 +721,7 @@ private static void createFileIfMissing(String filePath) {
     private static ArrayList<String> getLinesInFile(String filePath) {
         ArrayList<String> lines = null;
         try {
-            lines = new ArrayList(Files.readAllLines(Paths.get(filePath)));
+            lines = new ArrayList<>(Files.readAllLines(Paths.get(filePath)));
```

Comment:
Good practice to always include the type declaration so that compiler does not complain.